src/main/java/edu/greenblitz/tobyDetermined/commands/intake/Roller/RollerCommand.java

```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.tobyDetermined.commands.intake.Roller;
+
+import edu.greenblitz.tobyDetermined.subsystems.intake.IntakeRoller;
+import edu.greenblitz.utils.GBCommand;
+
+public abstract class RollerCommand extends GBCommand {
+	IntakeRoller roller;
+	RollerCommand(){
+		super();
```

Comment:
Not a deal breaker, but just to let you know.
This line is being called by default, you don't have to write this explicit. (In general, when you don't call a super constructor in the first line, super() is called by deafult.)

Reply:
DONE